src/commands/run.rs

```diff
@@ -164,17 +165,17 @@ pub(crate) fn do_run(
         prehashed_cache_key,
     )?;
     debug!("Running command with args: {:?}", command_vec);
-    let mut child =
-        Command::new(&runtime)
-            .args(&command_vec)
-            .spawn()
-            .map_err(|e| -> failure::Error {
-                RunError::ProcessFailed {
-                    runtime: runtime,
-                    error: format!("{:?}", e),
-                }
-                .into()
-            })?;
+    let mut child = Command::new(&runtime)
+        .args(&runtime_args)
```

Comment:
Does passing `Vec<String>` work well here? and does passing an empty vec here as it does in the default case cause any problems?  Ideally we'd have a line or two in the tests for this!

Reply:
> Does passing Vec<String> work well here?

Yup, it worked localy!

> does passing an empty vec here as it does in the default case cause any problems?

The spec doesn't specify otherwise... so I think we will be fine!
https://doc.rust-lang.org/std/process/struct.Command.html#method.args